force-app/main/default/triggers/OrderTrigger.trigger

```diff
@@ -0,0 +1,5 @@
+trigger OrderTrigger on Order (after update) {
+    OrderTriggerHandler.createAssetRecord(Trigger.new, Trigger.oldMap);
```

Comment:
why you are not performing if check here for checking if trigger is after update or not for future prospective we can add more options in our trigger like after update, after insert, before insert for that it's a good practice to always perform check for these cases